### ISSUE TITLE
@see does not generate link to reference class/method

### ISSUE THREAD

**Issue by othercorey** [cakephp/cakephp-api-docs#104]
The attributes section under the method in the class listing does not generate a link for the referenced method in from the @see tag.

https://api.cakephp.org/3.7/class-Cake.Console.CommandCollection.html#_addMany

>See
>\Cake\Console\CommandCollection::add()

**Comment by markstory** [comment]
We could probably separate the `see` annotation into a separate block like https://github.com/cakephp/cakephp-api-docs/blob/8139b106f59e3118525c0fc45bf248961fff396a/templates/cakephp/class.latte#L374-L381

We might be able to use the `classUrl` filter to make a link? It might get hard if the see tag contains other text though.

**Comment by othercorey** [comment]
I think the parser will throw an error if @see isn't a valid reference.  It's hard to follow what apigen is doing from issues/etc, but it seems upstream packages are deprecating @link in favor of @see so @see could contain either a valid http link or a valid class reference.  I'm not sure if classUrl would be unset if it's an http link and so on.

**Comment by othercorey** [comment]
Ok, I'm pretty far off on what's expected/allowed in @see after looking through cakephp source more. It can be pretty much anything.  I don't know that we can check for each type well enough to generate an appropriate link.

**Comment by othercorey** [comment]
Took me a while to figure out where this is in api gen, but looks like the annotation filter is supposed to figure this out for us.  Seems there is something causing the buildUrl call to fail.

https://github.com/ApiGen/ApiGen/blob/85290e8086f2f40e4d560de09ffd115466379dc2/packages/Annotation/src/AnnotationSubscriber/SeeUsesCoversAnnotationSubscriber.php#L66-L72

**Comment by ADmad** [comment]
@othercorey This is the repo which is used to generate our API docs: https://github.com/cakephp/cakephp-api-docs.

It's a forked of an old dev version of ApiGen. The ApiGen projecct has had a shaky life with development often stalling and restarting (and finally dead now) so we have just stuck with whats working for us in general, except for few issue like you have found.

**Comment by othercorey** [comment]
I noticed the issue talking about api gen being dead so I completely understand.  I thought this would be an easy win, but looks like it isn't :(.

**Comment by othercorey** [comment]
Short follow up for anyone searching.  It seems the attribute filter does generate a URL for a class types, but does not for method types.  I actually found an example that contains both.

https://api.cakephp.org/3.7/class-Cake.Database.Query.html#_orWhere
